### DISCUSSION TITLE
[Feature Request] Add package version to Docker tag

### DISCUSSION THREAD

**Discussion by bkaid** [DannyDainton/newman-reporter-htmlextra#364] [Ideas]
**Is your feature request related to a problem? Please describe.**
The pre-built Docker image is quite handy. It would be great if it also was tagged each release with it's version number so we can more easily control what version we use and to be able to tell what version of the package the Docker image includes.

**Describe the solution you'd like**
Docker image `dannydainton/htmlextra:latest` to be published with the same version number as the npm package in addition to `latest`.

**Describe alternatives you've considered**
I checked Docker Hub to confirm what's been published.

**Additional context**
Docker doesn't consider relying on `latest` a [best practice](https://docs.docker.com/develop/dev-best-practices/) .

**Comment by DannyDainton** [comment]
I've not looked at this since it was raised, not going to lie, docker isn't something I'm 100% comfortable with so I'm probably not going to be doing this.

Any help or advice would be very welcome.
